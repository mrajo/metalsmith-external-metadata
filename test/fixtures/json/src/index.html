<!doctype html>
<html>
	<head>
		<title>{{ title }}</title>
	</head>
	<body>
		<h1>{{ title }}</h1>
		<ul>
			{% for fruit in fruits %}
			<li>{{ fruit }}</li>
			{% endfor %}
		</ul>
	</body>
</html>