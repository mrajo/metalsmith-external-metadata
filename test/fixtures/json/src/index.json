{
	"title": "Testing JSON",
	"fruits": [
		"apple",
		"banana",
		"cherry",
		"durian"
	]
}