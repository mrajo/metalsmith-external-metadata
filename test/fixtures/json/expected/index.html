<!doctype html>
<html>
	<head>
		<title>Testing JSON</title>
	</head>
	<body>
		<h1>Testing JSON</h1>
		<ul>
			
			<li>apple</li>
			
			<li>banana</li>
			
			<li>cherry</li>
			
			<li>durian</li>
			
		</ul>
	</body>
</html>